training/src/anemoi/training/config/training/default.yaml

```diff
@@ -3,6 +3,7 @@ run_id: null
 fork_run_id: null
 load_weights_only: null # only load model weights, do not restore optimiser states etc.
```

Comment:
Guess here we would need just on `load_weights_only` and then also for consistency `transfer_learning` would also need to be defined as bool and added to the TrainingSchema?